ConcernsCaseWork/ConcernsCaseWork.Service.Tests/MeansOfReferral/MeansOfReferralServiceTests.cs

```diff
@@ -37,7 +38,7 @@ public async Task WhenGetMeansOfReferral_ReturnsMeansOfReferral()
 			httpClientFactory.Setup(_ => _.CreateClient(It.IsAny<string>())).Returns(httpClient);
 			
 			var logger = new Mock<ILogger<MeansOfReferralService>>();
-			var service = new MeansOfReferralService(httpClientFactory.Object, logger.Object, Mock.Of<ICorrelationContext>());
+			var service = new MeansOfReferralService(httpClientFactory.Object, logger.Object, Mock.Of<ICorrelationContext>(), Mock.Of<IClientUserInfoService>());
```

Comment:
SUGGESTION:

So I think we should potentially look at making the abstract service accessible through composition instead of inheritance

That way we wouldn't have to change every single file, each time it changes, because the class that uses it, doesn't need to know how it is constructed

Reply:
Ideally I think we would wrap the http client so that it's actually a different thing to the service classes, and compose it in. Headers wouldn't have been much of an issue then.
The inheritance was actually already in place early in the project